File: web3rpc/api/code-samples/python/debug/profiling/writeBlockProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "block.profile"

sdk = OpenSDK(host)
debug_response = sdk.debug.write_block_profile(fileName)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/writeBlockProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const file = "block.profile";

    provider.debug.writeBlockProfile(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()